aup.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Acceptable Use Policy"
+---
+
+The terms "R-multiverse", "User", "Contributor", "Moderator" and "Administrator" shall have the meaning as defined in the [R-multiverse Governance Document](governance.md).
+This Acceptable Use Policy (the "Policy") outlines the standards of conduct expected from all Users and Contributors to R-multiverse (the "Repository").
+By accessing or using the Repository, you agree to comply with this Policy.
+Violations of this Policy may result in the suspension or termination of your access to the Repository.
+
+## 1. Prohibited Content
+
+Users and Contributors must not submit, post, or share any content that:
+
+- **Illegal or Unlawful**: Violates any applicable laws, regulations, or guidelines in your jurisdiction, including content that promotes illegal activities, constitutes fraud, or violates intellectual property rights.
```

Comment:
what is "your jurisdiction"

Reply:
Your legal jurisdiction i.e. the laws that apply to the user, as in e.g. https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#c-acceptable-use
Because the concept of what's illegal differs across geographies, it may create enforcement problems if we specify a specific jurisdiction is what I understand.

Reply:
See my comment above. I'm not a lawyer so I can't answer your question. I'm just taking GitHub precedent (as per the link). We shouldn't be inventing anything here - this is all very tried and tested.

Reply:
GitHub's wording is "Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction." The "your jurisdiction" part is provided as an example, whereas the "applicable laws" part is the actual rule. If we do use "in your jurisdiction", I think it should be for illustrative purposes only so we don't create loopholes. Suggestion:

```
- **Illegal or Unlawful**: content that promotes illegal activities, constitutes fraud, violates intellectual property rights, or violates any applicable laws in any other way, including but not limited to the regulations, laws, and guidelines in your jurisdiction.
```

Reply:
I've amended in [9fe0b2d](https://github.com/r-multiverse/r-multiverse.github.io/pull/29/commits/9fe0b2d2bbcc8060f5185b42ee29e0c4488ec9bf) along the lines of your suggestion.